### PR TITLE
graph-service to handle GraphQL queries

### PR DESCRIPTION
**Pull request by jachro**
This PR introduces changes required for a new graph-service providing GraphQL endpoint. It also brings relevant documentation changes.

### CODE REVIEW

File: charts/renku/requirements.yaml [hunk 1]
```diff
@@ -17,8 +17,8 @@ dependencies:
   condition: gitlab.enabled
 - name: renku-graph
   alias: graph
-  repository: "https://swissdatasciencecenter.github.io/helm-charts/"
-  version: "0.1.0"
+  repository: "file://../../../renku-graph/helm-chart/renku-graph"
```
Comment (lorenzo-cavazzi) [inline charts/renku/requirements.yaml]: This probably sneaked in :smile:

Comment (jachro) [reply charts/renku/requirements.yaml]: Thanks Lorenzo. I fixed that

**Comment by lorenzo-cavazzi** [comment]
Reviewed and squashed, it's ready to be merged but I can't. Any @SwissDataScienceCenter/renku-maintainers willing to do this? :slightly_smiling_face:

File: charts/renku/values.yaml [hunk 2]
```diff
@@ -613,3 +613,9 @@ graph:
       ## A secret for signing request header tokens to be sent by GitLab with the Push Events
       ## Generated using: `openssl rand -hex 8|base64`
       secret: 1234
+
+  knowledgeGraph:
+    services:
+      renku:
+        ## Renku UI base url e.g. https://dev.renku.ch
+        url: https://dev.renku.ch
```
Comment (rokroskar) [inline charts/renku/values.yaml]: This value should be blank by default

```suggestion
        url: 
```

**Pull request merged**